ouroboros-consensus-cardano/ouroboros-consensus-cardano.cabal

```diff
@@ -572,7 +572,7 @@ library unstable-cardano-tools
     sop-extras,
     strict-sop-core,
     text,
-    text-builder,
+    text-builder >=0.6.10,
```

Comment:
I would have left a comment on this change, but the CI cabal formatting stuff wanted me to move the comment to just under the `build-depends` where it would have made zero sense.